src/aere_parse.erl

```diff
@@ -64,6 +65,11 @@ commands() ->
         , "cannot be adjusted using the put function."
         , "Cleans user variables and functions."]
        }}
+    , {"event",
+       {["ev"], consume, "SOPHIA_TYPE",
+        [ "Changes the in-REPL event type. Expects a Sophia variant type."
+        , "Cleans user variables, functions, typedefs, type scopes, and older events."]
```

Comment:
Comment by radrow copied from an older PR:

User is not aware of the difference between typedefs and typescopes. Just say "types".

Also, I think it's clearer to say "previously emitted events"